tests/testGetInstcal.py

```diff
@@ -85,9 +85,6 @@ def testInstcal(self):
         self.assertAlmostEqual(weather.getAirPressure(), visit229388_info['weath_airPressure'])
         self.assertAlmostEqual(weather.getHumidity(), visit229388_info['weath_humidity'])
 
-        # Example of metadata which should *not* have been copied from zeroth extension.
-        self.assertNotIn("PROPOSER", exp.getMetadata().paramNames())
```

Comment:
Ok. That's probably all best tested as part of the general set of metadata tests in obs_base (which obs_decam is already partly ported to on DM-7730).